Sources/MapboxNavigation/Resources/Assets.xcassets/AnnotationCentered.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "AnnotationCentered.png",
```

Comment:
These assets should be stored as scalable PDFs. #2734/#2873 introduced more stylish RouteInfoAnnotationLeftHanded and RouteInfoAnnotationRightHanded assets, but there’s no corresponding RouteInfoAnnotationCentered asset.